deploy/files/systemd/gunicorn.service

```
@@ -5,7 +5,7 @@ After=network.target
 User=wcivf
 Group=wcivf
 WorkingDirectory=/var/www/wcivf/code/
-ExecStart=/bin/bash -c 'PATH=/var/www/wcivf/venv/bin/:$PATH exec /var/www/wcivf/venv/bin/gunicorn \
+ExecStart=/bin/bash -c 'PATH=/var/www/wcivfcode/.venv/bin/:$PATH exec /var/www/wcivf/code/.venv/bin/gunicorn \
```

Hmm. This looks like it should be

```suggestion
ExecStart=/bin/bash -c 'PATH=/var/www/wcivf/code/.venv/bin/:$PATH exec /var/www/wcivf/code/.venv/bin/gunicorn \
```

---

Ok, deployed and tested again. I'll leave that deploy up in case you want to take a look